playbooks/02-infra.yml

```diff
@@ -86,6 +86,22 @@
       ansible.builtin.include_role:
         name: cert_manager
 
+    # Deploy Sushy Emulator into OCP
+    - name: Deploy Sushy Emulator service pod
+      when: cifmw_use_sushy_emulator | default(false) | bool
+      vars:
+        cifmw_sushy_emulator_hypervisor_address: "{{ hostvars['controller'].ansible_host }}"
+        cifmw_sushy_emulator_hypervisor_target: controller
+        cifmw_sushy_emulator_install_type: ocp
+      ansible.builtin.include_role:
+        name: sushy_emulator
+
+    - name: Setup Libvirt on controller
+      when: cifmw_use_libvirt_extracted | default(false) | bool
```

Comment:
(non-blocking) question: Make sense to "Setup libvirt on controller" when cifmw_use_sushy_emulator is missing or false?

On the top of my head, I think if both are required for support deploying Sushy if the first is not applicable, the second should not be applied as well.

Reply:
Hi @evallesp, 

Yes, I think this is a case of "YAGNIi"

I was just adding the option for other use cases outside this Ironic/Sushy one to setup the controller node as a hypervisor but I can't even imagine now what that would be so I will lessen the scope.

Reply:
Done